Libplanet.Extensions.Cocona/Commands/StoreCommand.cs

```diff
@@ -26,7 +27,58 @@ public class StoreCommand
                     ["monorocksdb"] = storePath => new MonoRocksDBStore(storePath),
                 }.ToImmutableSortedDictionary();
 
-        [Command(Description = "Query a block by index")]
+        [Command(Description = "Build an index for transaction id and block hash.")]
+        public void BuildIndexTxBlock(
+            [Argument("STORE", Description = StoreArgumentDescription)]
+            string home,
+            [Argument("OFFSET", Description = "block index")]
+            int offset,
+            [Argument("LIMIT", Description = "block index")]
+            int limit
+        )
+        {
+            var store = LoadStoreFromUri(home);
```

Comment:
We prefer specifying type explicitly if code-reviewers can't guess its type from a statement. 
```suggestion
            IStore store = LoadStoreFromUri(home);
```

Reply:
I prefer to use `var` instead of declaring type explicitly for static type language. It looks more readable for me.
But as you mentioned, for some statements like function calls, it is not straightforward for readers to guess the type of variable.

So I conclude that some statements like function calls would be better to be declared with Type.

Thanks for your advice.